week-4/F-object-keys/exercise-part-0.js

```diff
@@ -0,0 +1,32 @@
+/*
+
+Return the keys of the following object
+
+*/
+
+var capitalCities = {
+  edinburgh: 'Scotland',
+  nairobi: 'Kenya',
+  canberra: 'Australia',
+  ottowa: 'Canada'
+};
+
+var highScores = {
+  55: 'Alistair',
+  100: 'David',
+  89: 'Hannah',
+  34: ['Sergi', 'Frank',]
+};
```

Comment:
Hmm, is there any other example we could use here?

The way I would think to store a list of high scores would be in a sorted array, rather than an object of score, mapped to names:

```js
// I'd make sure to not allow the array to grow bigger than 10, or the max number of max scores I want to keep
var highScores = [
 {name: 'Alistair' , score: 100},
 {name: 'Hanna', score: 78}
//etc.
]
```